phpstan-baseline.neon

```diff
@@ -8290,6 +8290,11 @@ parameters:
 			count: 1
 			path: app/Models/Application.php
 
+		-
+			message: "#^PHPDoc type array of property App\\\\Models\\\\ApplicationMetrics\\:\\:\\$primaryKey is not covariant with PHPDoc type string of overridden property Illuminate\\\\Database\\\\Eloquent\\\\Model\\:\\:\\$primaryKey\\.$#"
```

Comment:
Not allowed to add stuff to this file

Reply:
hi @Jellyfrog 
if you look at table "application_metrics" you will see that there is composite primary key
`UNIQUE INDEX `application_metrics_app_id_metric_unique` (`app_id`, `metric`) USING BTREE`

only existing  App/Models with two primaryKeys is UserPref.php
and it is listed in "phpstan-baseline.neon" exactly from this reason
`    protected $primaryKey = ['user_id', 'pref'];`

without this, error is thrown on check
so i make same as UserPref.php

if you have any other suggestion ? how to solve this other way ?

Reply:
error is about $primaryKey is array in my case

`PHPDoc type array of property App\Models\ApplicationMetrics::$primaryKey is not covariant with PHPDoc type string of overridden property Illuminate\Database\Eloquent\Model::$primaryKey. `